weaver/wps_restapi/processes/processes.py

```diff
@@ -216,11 +219,13 @@ def execute_process(self, job_id, url, headers=None, notification_email=None):
                         job.status_message = "Job failed{}.".format(msg_progress)
                         wps_package.retrieve_package_job_log(execution, job)
                         job.save_log(errors=execution.errors, logger=task_logger)
+                    task_logger.debug("Mapping Job references with generated WPS locations.")
+                    map_locations(job, settings)
```

Comment:
You map what here? The current job is monitored by PyWPS, so, if the job run a local PyWPS process (only case I see is for workflows?) you link directly the job process status to the api process status?

Reply:
In the wps-outputs, the job-id, and pywps 'process-execution-id' don't match. 
Technically, there is not problem, you query the API route `/jobs/{jobId}/results`, and each path in the outputs have `{wps-outputs-path}/{processExecId}/{the-output}`. It is the same of the xml status and log files generated. They have the name based on `{processExecId}`. 
The map only attempts to create a symlink such that `{jobId}->{processExecId}`. 
This is not a 'functional' requirement, but more a 'for convenience' one. 
Avoids me going through hundreds of job logs to find the other ID to do the mapping manually.

Reply:
Example
![image](https://user-images.githubusercontent.com/19194484/83443805-54607780-a418-11ea-9d1f-534164402054.png)

Reply:
Clever, I remember digging into that stuff!